### ISSUE TITLE
Prevent flash of text using CDNs for fonts

### ISSUE THREAD

**Issue by joelalejandro** [SomosCodear/codear-site#8]
As described in KyleAMathews/typefaces#128, self-hosted fonts incur in a rendering delay that's noticeable enough for our users.

I recommend we switch to the Google Fonts loader:

```
<link href="https://fonts.googleapis.com/css?family=Roboto:100,400,700|Source+Sans+Pro:400,700&display=swap" rel="stylesheet">
```

**Comment by joelalejandro** [comment]
Also, our default font should be `Source Sans Pro`. `Roboto` is sparingly used in comparison.

**Comment by asermax** [comment]
Fixed on e77b65b33027d1388d2699bad3230a2048ca76f0
